gamemodes/cinema/gamemode/modules/theater/services/sh_dlive.lua

```diff
@@ -15,93 +15,17 @@ function SERVICE:GetKey(url)
 end
 
```

Comment:
I don't think this service needs to be maintained since no one uses it anymore, but it's up to you

Reply:
yea I know but I already wrote it